### ISSUE TITLE
Grunt hits ENOTSUP on windows network paths due to outdated glob dependency

### ISSUE THREAD

**Issue by xirzec** [gruntjs/grunt#1401]
Per isaacs/node-glob#205 you'll see grunt fail with a "ENOTSUP" when scanning network directories for globs that have a ** in them. 

Easy fix, just update the glob dependency to 5.0.15.

**Comment by kitsguru** [comment]
What is the best way to do this? 

I have grunt installed locally which has a glob dependency of "glob": "~3.1.21",

The local version of glob is 5.0.14 with "_from": "glob@>=5.0.14 <6.0.0", under node_modules but glob is also installed in grunt/node_modules as version 3.1.21

Global version of glob installed is currently 6.0.1

Running npm install glob results in the local version of glob being 6.0.1 but the grunt appears to still be using 3.1.21. 

I am unsure of the best way to fix this. Any advice is appreciated.

**Comment by xirzec** [comment]
Here's what I did as a workaround:

1. npm shrinkwrap
2. Manually update the glob dependency under grunt in the shrinkwrap file
3. Either delete node_modules and reinstall or go into `node_modules\grunt\node_modules` and delete glob.

**Comment by kitsguru** [comment]
Shrinkwrap failed the first time I ran it. I deleted the entire node_modules and re-installed all the dev dependencies. Since grunt is a devdependency it skipped grunt.

I then applied your workaround and it worked ok. 

**NOTE**
I see that some of the dependencies for grunt have been updated in the last couple of days but not the glob - still at 3.1.21 whereas 6.0.1 is the latest and it fixes this issue.
